src/aca_main.cpp

```diff
@@ -52,6 +52,8 @@ string g_grpc_server_port = EMPTY_STRING;
 string g_ofctl_command = EMPTY_STRING;
 string g_ofctl_target = EMPTY_STRING;
 string g_ofctl_options = EMPTY_STRING;
+string g_NCM_ADDRESS = EMPTY_STRING;
+string g_NCM_PORT = EMPTY_STRING;
```

Comment:
to make it 100% consistant let's change:
g_NCM_ADDRESS -> g_ncm_address
g_NCM_PORT -> g_ncm_port

Reply:
fixed